src/main/comments.js

```diff
@@ -540,12 +539,19 @@ function printDanglingComments(path, options, sameIndent, filter) {
   return indent([hardline, join(hardline, parts)]);
 }
 
-function printCommentsSeparately(path, options) {
+function printCommentsSeparately(path, options, skip) {
```

Comment:
`skip` is a verb. Only functions should be named with verbs. How about `ignored`?